sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -261,6 +261,17 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val HADOOPFSRELATION_SIZE_FACTOR = buildConf(
+    "org.apache.spark.sql.execution.datasources.sizeFactor")
+    .internal()
+    .doc("The result of multiplying this factor with the size of data source files is propagated" +
+      " to serve as the stats to choose the best execution plan. In the case where the " +
+      " the in-disk and in-memory size of data is significantly different, users can adjust this" +
```

Comment:
nit: always put space at the end of the line for readability / consistency.
we have two spaces here
```
"...In the case where the " +
" the in-disk and in-..."
```

Reply:
and double `the`

Reply:
actually https://github.com/apache/spark/pull/20072/files/ec275a841a7bb4c23b277f915debeed54e6cf7ea#diff-9a6b543db706f1a90f790783d6930a13R250 is missing a space at the end - could you also fix that

Reply:
done, thanks